src/components/modules/ui.ts

```diff
@@ -392,8 +392,12 @@ export default class UI extends Module {
    * @param {KeyboardEvent} event - keyboard event
    */
   private defaultBehaviour(event: KeyboardEvent): void {
-    const keyDownOnEditor = (event.target as HTMLElement).closest(`.${this.CSS.editorWrapper}`);
     const { currentBlock } = this.Editor.BlockManager;
+    /**
+     *  currentBlock !=undefined
+     *  Ignore when KeyDown Element is null
+     */
+    const keyDownOnEditor = (event.target as HTMLElement).closest(`.${this.CSS.editorWrapper}`) || currentBlock != undefined;
```

Comment:
let's move the `currentBlock !== undefined` statement to the separate value and describe this case (why it is counts as keydown on editor)

Reply:
Done